TTS/utils/arguments.py

```diff
@@ -67,11 +72,11 @@ def parse_arguments(argv):
     return parser.parse_args()
 
 
-def get_last_checkpoint(path):
-    """Get latest checkpoint from a list of filenames.
+def get_last_models(path):
+    """Get latest checkpoint or/and best model in path.
```

Comment:
For the docstrings we were using https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
It's more compact :+1: These are the numpy ones that Spyder automatically generates. I'll redo them.

Reply:
sry for the rework :(

Reply:
As a side note - are the circleci test runs free? If not maybe we can only do them once the PR is complete?

Reply:
It is not something I'm worried about so far :) but thx for the flag.

Reply:
I think keeping the function name as before is more intuitive. In the end best model is also a checkpoint.